cBioPortal/cbioportal-frontend
# Generic assay frontend implementation

Fix cBioPortal/cbioportal#6943
Fix cBioPortal/cbioportal#6366

Describe changes proposed in this pull request:
- a make all generic assay data available in the oncoprint tab as heatmap track
![image](https://user-images.githubusercontent.com/15748980/75392876-efe2c900-58ba-11ea-9850-99146ee41fa3.png)

    tooltip (link and description)
![image](https://user-images.githubusercontent.com/15748980/75462062-a1ccd480-5951-11ea-8e4b-0fe0618754c7.png)

- b make all generic assay data available in the plots tab
![image](https://user-images.githubusercontent.com/15748980/75393047-46e89e00-58bb-11ea-8b1e-d7d200da5be2.png)

    Entities selected from the oncoprint (added heatmap tracks) will list as the top options:
![image](https://user-images.githubusercontent.com/15748980/75463911-73042d80-5954-11ea-8a37-adb1abb87e95.png)

    info icon (link and description)
![image](https://user-images.githubusercontent.com/15748980/75461953-7ba73480-5951-11ea-96f5-cc0daf823c34.png)

**Please Note**
The `mutational signatures` data is not the final data, we are still in the process of deciding the `name` and `description` of each signature. Ref: https://github.com/cbioportal/cbioportal/issues/7198
**How to test:**
1. go to https://triage.cbioportal.mskcc.org and select `msk_impact_2017` study.
2. press `F12` to open the browser console and paste this into console then enter: `localStorage.setItem("netlify", "deploy-preview-3025--cbioportalfrontend")`
3. check the new features in oncoprint tab (result view),  plots tab(result view).
